tools/js-sdk-release-tools/src/common/rushUtils.ts

```diff
@@ -89,8 +89,9 @@ export async function buildPackage(
 
     // build sample and test package will NOT throw exceptions
     // note: these commands will delete temp folder
-    await tryBuildSamples(packageDirectory, rushxScript);
+    await tryBuildSamples(packageDirectory, rushxScript);    
     await tryTestPackage(packageDirectory, rushxScript);
+    await tryFormatSamples(packageDirectory);
```

Comment:
the command format src, test and sample code
```suggestion
    await tryFormatSdk(packageDirectory);
```